### ISSUE TITLE
termux-call-log hangs

### ISSUE THREAD

**Issue by l0pu** [termux/termux-api-package#54]
I ran last version of termux-call-log on rooted galaxy s8+ and termux-call-log just hands without any output. Is there any fix?

**Comment by Oj18** [comment]
Please post the output of `termux-info` so I can attempt to help further.

Thanks.

**Comment by meshr-net** [comment]
it works now. I installed last version.

**Comment by Oj18** [comment]
You don't appear to be the OP?

**Comment by l0pu** [comment]
it works now after installing the last version. BTW i followed 'Backup and restore Termux installation' instruction and lost my home directory:( One should backup /data/data/com.termux/files/ instead of just $PREFIX to keep  home directory

**Comment by Oj18** [comment]
Yeah but it might work for you but it still might not work for the person who originally made the issue. Unless you're an alt..?
